Makefile

```diff
@@ -24,7 +24,7 @@ test_integration_quick ::
 		rspec -t quick
 
 # TODO: add race
-checks :: vet fmt lint climate race
+checks :: vet fmt lint race climate
```

Comment:
The intention is to see which tests fail before calculating coverage. If certain tests fail, the coverage drops and if it drops below the threshold we have no clue why. I believe we could also fix this in `scripts/climate`.